src/components/Homepage/blog.js

```diff
@@ -18,28 +18,29 @@ const Blog = () => (
       <LatestPosts>
         {posts => (
           <ul>
-            {posts.slice(0, 3).map(({ node }) => (
-              <li key={`${node.id}`}>
-                <Subtitle noPaddingBottom>
-                  <ExternalAnchor
-                    href={`https://medium.com/yld-engineering-blog/${
-                      node.uniqueSlug
-                    }`}
-                  >
-                    {node.title}
-                  </ExternalAnchor>
-                </Subtitle>
-                <BodyPrimary noPaddingTop>
-                  {format(new Date(node.createdAt), 'MMMM DD[,] dddd')}
-                </BodyPrimary>
-                <Hr />
-              </li>
-            ))}
+            {posts
+              .slice(0, 3)
+              .map(({ node }) => node)
+              .map(({ id, uniqueSlug, title, firstPublishedAt }) => (
+                <li key={`${id}`}>
+                  <Subtitle noPaddingBottom>
```

Comment:
this homepage blog section (which should be shared on a component with the speciality page) had subtle spacing and interaction differences. I fixed that, but we should make a component out of this

Reply:
I know the name of the ticket (styling/states) doesn't seem to imply this, but I reckon we should have somewhere to keep track of these things like https://trello.com/c/AymuwRmi/307-styling-re-work-to-match-designed-stateful-components ?

Reply:
yeah, I'll add it there

Reply:
![image](https://user-images.githubusercontent.com/3236388/55489799-8e729580-562a-11e9-9500-c69abc2b3cca.png)